starter-code/src/functions-and-arrays.js

```diff
@@ -10,14 +17,44 @@ var words = [
   'orchard',
   'crackpot'
 ];
+function findLongestWord(array) {
+  var longestWord;
+  if(array.length!==0){
+    var maxLength=array[0].length;
+    longestWord=array[0];
+    array.forEach(function(word){
+      if(word.length>maxLength){
+        maxLength=word.length;
+        longestWord=word;
```

Comment:
Here you could initialize the longestWord var as an empty string and then compare that length with the one with the word you are checking in the array